releases/release-7.1.0.md

```diff
@@ -233,14 +367,196 @@ In v7.1.0, the key new features and improvements are as follows:
+    - Improve TiFlash performance and stability in the disaggregated storage and compute architecture [#6882](https://github.com/pingcap/tiflash/issues/6882) @[JaySon-Huang](https://github.com/JaySon-Huang) @[breezewish](https://github.com/breezewish) @[JinheLin](https://github.com/JinheLin)
     - Support optimizing query performance in Semi Join or Anti Semi Join by selecting the smaller table as the build side [#7280](https://github.com/pingcap/tiflash/issues/7280) @[yibin87](https://github.com/yibin87)
+    - (dup) Improve performance of data import from BR and TiDB Lightning to TiFlash with default configurations [#7272](https://github.com/pingcap/tiflash/issues/7272) @[breezewish](https://github.com/breezewish)
+
++ Tools
+
+    + Backup & Restore (BR)
+      <!-- **tw:hfxsd** (1) -->
+        - Support for online modification of `tikv.log-backup.max-flush-interval` configuration [#14433](https://github.com/tikv/tikv/issues/14433) @[joccau](https://github.com/joccau)
+
+    + TiCDC
+      <!-- **tw:ran-huang** (6) -->
+        - Optimized the directory structure when DDL occurs under the scenario of synchronization to object storage [#8890](https://github.com/pingcap/tiflow/issues/8890) @[CharlesCheung96](https://github.com/CharlesCheung96)
+        - Optimized the method of setting GC TLS for upstream when CDC's synchronization task fails [#8403](https://github.com/pingcap/tiflow/issues/8403) @[charleszheng44](https://github.com/charleszheng44)
+        - Added support for synchronization to KOP downstream [#8892](https://github.com/pingcap/tiflow/issues/8892) @[hi-rustin](https://github.com/hi-rustin)
+        - When synchronizing to Kafka, it supports using the open-protocol protocol to only synchronize columns that have changed when an update occurs [#8706](https://github.com/pingcap/tiflow/issues/8706) @[sdojjy](https://github.com/sdojjy)
+        - Optimized the way CDC handles errors when downstream faults and other scenarios occur [#8657](https://github.com/pingcap/tiflow/issues/8657) @[hicqu](https://github.com/hicqu)
+        - Added a configuration to control whether to set the authentication algorithm in the scenario of enabling TLS [#8867](https://github.com/pingcap/tiflow/issues/8867) @[hi-rustin](https://github.com/hi-rustin)
+
+    + TiDB Lightning
+      <!-- **tw:hfxsd** (3) -->
+        - Change the Region distribution precheck item from Critical to Warn [#42836](https://github.com/pingcap/tidb/issues/42836) @[okJiang](https://github.com/okJiang)
+        - Add retry for unknown RPC errors during import [#43291](https://github.com/pingcap/tidb/issues/43291) @[D3Hunter](https://github.com/D3Hunter)
+        - Enhance Region job retry [#43682](https://github.com/pingcap/tidb/issues/43682) @[lance6716](https://github.com/lance6716)
```

Comment:
```suggestion
        - Change the severity level of the precheck item related to uneven distribution of regions from `Critical` to `Warn` to avoid blocking users from importing data [#42836](https://github.com/pingcap/tidb/issues/42836) @[okJiang](https://github.com/okJiang)
        - Add retry after unknown RPC errors occur during data import [#43291](https://github.com/pingcap/tidb/issues/43291) @[D3Hunter](https://github.com/D3Hunter)
        - Enhance a retry mechanism for region jobs [#43682](https://github.com/pingcap/tidb/issues/43682) @[lance6716](https://github.com/lance6716)
```